### ISSUE TITLE
Problem with '(instancetype)'

### ISSUE THREAD

**Issue by Tedko** [drkameleon/iswift#60]
code below will trigger the warning:

```objective-c
@implementation ABC
-(instancetype)initWithFrame:(CGRect)frame

```

however, it could compile in Xcode

**Comment by drkameleon** [comment]
Hmmm... I think I know what it is....

Did it complain about an "Unknown type" for `instancetype`? Does it work if you replace `instancetype` with something else, e.g. `id`?

I'm on to it... ;)

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 1.8.
